### PR TITLE
fix label error

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR fixes a label error in the gating graphs.

### CODE REVIEW

File: figures/S3.supplemental_figure3/figures/final_figure_pt2.png
Comment (d33bs) [inline figures/S3.supplemental_figure3/figures/final_figure_pt2.png:1]: Are the dimensions and DPI settings for this visualization tied to a specific documented requirement (for instance, from a journal, or other platform)? If so, consider adding documentation to support the decision-making surrounding this for the code which generated the visualization.

**Review by d33bs — COMMENTED** [review]
Nice work! I left a few comments and suggestions - please don't hesitate to let me know if you have any questions.

File: figures/S3.supplemental_figure3/figures/final_figure_pt2.png
Comment (MikeLippincott) [reply figures/S3.supplemental_figure3/figures/final_figure_pt2.png:1]: No specific requirements - just general rules of thumb for figures. These can easily be changed and are in someways arbitrary.

[the remark this answers is not in the document]
Comment (MikeLippincott) [reply figures/S3.supplemental_figure3/figures/final_figure_pt2.png:1]: Addressed

**Pull request merged**